File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/main/java/simulacionBandaMusica/factories/BajistaFactory.java
package simulacionBandaMusica.factories;

import simulacionBandaMusica.models.Bajista;

import static simulacionBandaMusica.utils.sorteoDatos.*;

public class BajistaFactory {

    public static Bajista create() {
        String name = randomName();
        int experienceYear = randomAnno();
        int numCuerdas = randomNumCuerdas();
        return new Bajista(name, "Bajo", experienceYear, numCuerdas);
    }

}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/main/kotlin/simulacionBandaMusica/factories/BajistaFactory.kt
package simulacionBandaMusica.factories

import simulacionBandaMusica.models.Musico
import simulacionBandaMusica.utils.sorteoDatos

object BajistaFactory {
    fun create(): Musico.Bajista {
        val name = sorteoDatos.randomName()
        val experienceYear = sorteoDatos.randomAnno()
        val numCuerdas = sorteoDatos.randomNumCuerdas()
        return Musico.Bajista(name, "Bajo", experienceYear, numCuerdas)
    }
}